WORKSPACE

```diff
@@ -1 +1,84 @@
 workspace(name = "com_github_buildbarn_bb_deployments")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "com_github_buildbarn_bb_storage",
+    sha256 = "5ba46369868c07e5600584fb0efed42be6596f0da8940495771990070a6a6aab",
```

Comment:
Even though I am generally a fan of `http_archive()` over `git_repository()`, we'd better not do it for repos that don't have releases. There is no guarantee that the SHA sum of the tarball is stable.